src/xdocs/config_coding.xml

```diff
@@ -4468,6 +4468,24 @@ enum Foo10 { FOO, BAR } // OK
         <source>
 &lt;module name=&quot;NoFinalizer&quot;/&gt;
         </source>
+        <p>Example:</p>
+          <div class="wrapper">
+              <pre>
```

Comment:
Don't use these tags here. Please take a look at other pull requests and update it accordingly.

Reply:
done; i removed only div, i wrote div as it was mentioned in the example only.

Reply:
tried to remove div & pre, it gave me an error (as expected), tried to remove div only, i got an error:
![image](https://user-images.githubusercontent.com/51538094/79610643-36091b80-80f9-11ea-9aee-e2933209c5e7.png)

Reply:
`<source>` tag should be used instead

Reply:
done